<launch>
	<node pkg="mantis_interface_dynamixel" type="interface_node" name="dynamixel_interface" clear_params="true" output="screen">
		<param name="topic_input_setpoints" value="setpoints" type="str" />
		<param name="topic_output_states" value="states" type="str" />

		<param name="port_name" value="/dev/ttyUSB0" type="str" />
		<param name="port_baud" value="3000000" type="int" />
		<param name="protocol" value="2.0" type="double" />

		<param name="frame_id" value="mantis" type="str" />
		<param name="update_rate" value="50.0" type="double" />

		<!-- Motor Parameters -->
		<rosparam command="load" file="$(find mantis_interface_dynamixel)/launch/motor_config.yaml"/>
	</node>
</launch>
